alcali/package/install.sls

```diff
@@ -6,33 +6,33 @@
 {%- from tplroot ~ "/map.jinja" import alcali with context %}
 
 {% if alcali.config.db_backend == 'mysql' %}
-{% set db_connector = 'mysqlclient' %}
-{% set db_requirements = {
-    'RedHat': ['mariadb-devel', 'python3-devel'],
-    'Arch': ['mariadb-libs'],
-    'Debian': ['default-libmysqlclient-dev', 'python3-dev'],
-    'FreeBSD': ['curl'],
-}.get(grains.os_family) %}
+  {% set db_connector = 'mysqlclient' %}
+  {% set db_requirements = {
+      'RedHat': ['mariadb-devel', 'python3-devel'],
+      'Arch': ['mariadb-libs'],
+      'Debian': ['default-libmysqlclient-dev', 'python3-dev'],
+      'FreeBSD': ['curl'],
+  }.get(grains.os_family) %}
 {% elif alcali.config.db_backend == 'postgresql' %}
-{% set db_connector = 'psycopg2' %}
-{% set db_requirements = {
-    'RedHat': ['libpq-devel', 'python3-devel'],
-    'Arch': ['postgresql-libs'],
-    'Debian': ['libpq-dev', 'python3-dev'],
-    'FreeBSD': ['postgresql-libpqxx'],
-}.get(grains.os_family) %}
+  {% set db_connector = 'psycopg2' %}
+  {% set db_requirements = {
+      'RedHat': ['libpq-devel', 'python3-devel'],
+      'Arch': ['postgresql-libs'],
+      'Debian': ['libpq-dev', 'python3-dev'],
+      'FreeBSD': ['postgresql-libpqxx'],
+  }.get(grains.os_family) %}
 {% endif %}
 
 {% set venv_requirements = {
-    'RedHat': ['python3-virtualenv'],
+    'RedHat': ['python-virtualenv'],
```

Comment:
This breaks on CentOS 7/8 at least. Not sure where this even works? Fedora maybe?